beast/physicsmodel/stars/ezmist/mist.json

```diff
@@ -37,7 +37,7 @@
 		],
 	"Av_value": 0,
 	"defaults": {
-		"version": 1.0,
+		"version": 1.2,
```

Comment:
Does this change the outputs?  There is a failure in the travis testing.  What is the difference between versions 1 and 1.2?

Reply:
I checked this: no, this update does not change outputs.  In fact, that's why I updated the file: passing the incorrect version did nothing (no failure of webform submit), so I thought it would be best to update in case it does cause a failure in the future.  There's only one option available (1.2), so likely it is currently an unchecked dummy variable.